src/UnitTests/Projection/MoreExplanatoryExceptionTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using AutoMapper.QueryableExtensions;
+
+namespace AutoMapper.UnitTests.Projection
+{
+    public class MoreExplanatoryExceptionTests
+    {
+        [Fact]
+        public void ConstructorWithUnknownParameterTypeThrowsExplicitException()
+        {
+            var config = new MapperConfiguration(cfg =>
+            {
+                cfg.CreateMap<EntitySource, EntityDestination>();
+            });
+            var exception = Assert.Throws<BuilderException>(() => config.ExpressionBuilder.GetMapExpression<EntitySource, EntityDestination>());
```

Comment:
ProjectTo is the API, so the test should call that. GetMapExpression is just an internal thing.

Reply:
`ProjectTo` is now used by unit test.